docs/section10/10-14.md

```diff
@@ -0,0 +1,177 @@
+# Airside systems – Rule 10-14  
+**Schema Version:** 0.0.36        
+**Mandatory Rule:** True          
+**Rule ID:** 10-14  
+ 
+**Rule Description:** Baseline shall be modeled with the heating HVAC system efficiency per Tables G3.5.1-G3.5.6 (applies only to the heating efficiency of baseline furnaces and heat pumps).  Where multiple HVAC zones or residential spaces are combined into a single thermal block the heating efficiencies (for baseline HVAC System Types 3 and 4) shall be based on the  equipment capacity of the thermal block divided by the number of HVAC zones or residential spaces.   
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1.2.1 Equipment Efficiencies  
+**Appendix G Section Reference:** None  
+**Data Lookup:** Tables G3.5.2, G3.5.4 and G3.5.5  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**  
+
+1. Applicable to baseline HVAC system types 2, 3, 3a, 4, and 9.  
+
+**Function Call:**  
+
+1. get_baseline_system_types()  
+2. get_hvac_zone_list_w_area()  
+3. baseline_system_type_compare()  
+
+## Rule Logic:   
+**Applicability Check 1:**  
```

Comment:
The RCT will likely have all of the expected efficiency values stored separately in the path `RCT\rct229\rulesets\ashrae9012019\data` 
Should we structure the RMD similarly so that all expected efficiency values are organized together? There are more changes we could make to the RMD to have it be more concise and less repetitive but I'm not sure it's worth improving the RMD since the RCT dev team will likely write the logic differently anyway.

Example chiller efficiency data is stored as shown below.
{
  "prm_water_chiller_min_efficiency": [
    {
      "equipment_type": "displacement",
      "inclusive_minimum_capacity_tons": 0.0,
      "exclusive_maximum_capacity_tons": 150.0,
      "minimum_full_load_efficiency_kw_per_ton": 0.790,
      "minimum_integrated_part_load_kw_per_ton": 0.676
    },
    {
      "equipment_type": "displacement",
      "inclusive_minimum_capacity_tons": 150.0,
      "exclusive_maximum_capacity_tons": 300.0,
      "minimum_full_load_efficiency_kw_per_ton": 0.718,
      "minimum_integrated_part_load_kw_per_ton": 0.629
    },
    {
      "equipment_type": "displacement",
      "inclusive_minimum_capacity_tons": 300.0,
      "exclusive_maximum_capacity_tons": 9999.99,
      "minimum_full_load_efficiency_kw_per_ton": 0.639,
      "minimum_integrated_part_load_kw_per_ton": 0.572
    },

Reply:
Agree, both should be fail.

Reply:
It is not clear what you are suggesting? That I rewrite it?

Reply:
Christina and Jackson talked and are on the same page with the structure. Leave as is.